cypress/e2e/course/createEditAndDeleteCourse.feature

```diff
@@ -1,78 +1,79 @@
+@group-A
```

Comment:
Probably you have to update docs about tags and mention for both tags which you have added along with script